lib/cocoapods-rome/post_install.rb

```diff
@@ -50,6 +50,16 @@ def xcodebuild(sandbox, target, sdk='macosx')
   Pod::UI.puts "Copying frameworks to `#{destination.relative_path_from Pathname.pwd}`"
 
   destination.rmtree if destination.directory?
+
+  installer_context.umbrella_targets.each do |umbrella|
+    umbrella.specs.each do |spec|
+      consumer = spec.consumer(umbrella.platform_name)
+      file_accessor = Pod::Sandbox::FileAccessor.new(installer_context.sandbox_root, consumer)
+      frameworks += file_accessor.vendored_libraries
+      frameworks += file_accessor.vendored_frameworks
+    end
+  end
```

Comment:
Currently there's a problem when these aren't removed because the script crashes when trying to copy a framework that doesn't exist